generator/postgres/postgres_generator.go

```diff
@@ -56,6 +56,14 @@ func GenerateDSN(dsn, schema, destDir string, templates ...template.Template) er
 	defer db.Close()
 
 	fmt.Println("Retrieving schema information...")
+	return GenerateDB(db, schema, cfg.Database, destDir, templates...)
```

Comment:
The schema name and database name are swapped here, which is causing the tests to fail.